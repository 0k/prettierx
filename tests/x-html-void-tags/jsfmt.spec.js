// [prettierx merge update(s) from prettier@2.3.1 ...]
// with bogus attributes to preserve snapshot order
// XXX TBD not working with Vue after merge - ???
run_spec(__dirname, ["html", "angular"], { bogus1: true });
run_spec(__dirname, ["html"], { bogus2: true, htmlVoidTags: true });
run_spec(__dirname, ["angular"], { bogus3: true, htmlVoidTags: true });
