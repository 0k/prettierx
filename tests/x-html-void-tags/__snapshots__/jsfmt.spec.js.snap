// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input.html - {"bogus1":true} format 1`] = `
====================================options=====================================
bogus1: true
parsers: ["html", "angular"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<input type='text'>

=====================================output=====================================
<input type="text" />

================================================================================
`;

exports[`input.html - {"bogus2":true,"htmlVoidTags":true} format 1`] = `
====================================options=====================================
bogus2: true
htmlVoidTags: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<input type='text'>

=====================================output=====================================
<input type="text">

================================================================================
`;

exports[`input.html - {"bogus3":true,"htmlVoidTags":true} format 1`] = `
====================================options=====================================
bogus3: true
htmlVoidTags: true
parsers: ["angular"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<input type='text'>

=====================================output=====================================
<input type="text" />

================================================================================
`;
